lab-mongoose-movies/routes/index.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express');
+const router  = express.Router();
+
+router.get('/', (req, res, next) => {
+  res.render('celebrities');
+  Celebrity.find()
+  .then((celebritiesFromDb) => {
+   res.locals.celebrityList = celebritiesFromDb;
+    res.render ('celebrities');
+  })
+  .catch ((err) => {
+    next(err);
+    res.render ('error');
+  });
+});
+
+router.get ('/about', (res,req, next) => {
```

Comment:
You mixed up the order of parameters. It's `(req, res, next)`. That's why we are getting an error